### PR TITLE
Task Module: refactoring

### LINKED ISSUE #716: Refactor Task module to be consistent with rest of the UX
Refactor the following top-level modules.

- [ ] List 
- [ ] Details
- [ ] Launch
- [ ] CT

### PR DESCRIPTION
**Pull request by oodamien**
Tasks module:
- Refactoring
- Remove Bulk Import Page
- Remove Applications Page
- Fix UX lists

Resolves #716 

TODO:
- Counters tabulation update (After remove)
- Back action (back history)

### CODE REVIEW

File: ui/src/app/tasks/task-launch/task-launch.component.html [hunk 1]
```diff
@@ -1,17 +1,120 @@
...
-  </div>
-  <div class="col-md-12 text-left">
-    <button type="button" class="btn btn-default" (click)="back()">Back</button>
-  </div>
+<div *ngIf="taskDefinition$ | async as taskDefinition; else loading">
+  <form [formGroup]="form" class="form-horizontal" role="form" (ngSubmit)="submit(taskDefinition.name)" novalidate>
+    <div class="heading">
+      <div class="description">
+        <h1 class="no-user-selection"><h1>Launch task <strong>{{ taskDefinition.name }}</strong></h1></h1>
+        <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.</p>
```
Comment (ghillert) [inline ui/src/app/tasks/task-launch/task-launch.component.html]: Lorem ipsum???

File: ui/src/app/tasks/task-launch/task-launch.component.html [hunk 2]
```diff
@@ -1,17 +1,120 @@
...
+      </div>
+    </div>
+    <div class="builder">
+      <div class="row">
+        <div class="col-md-12">
+          <div class="col-1-layout" formArrayName="args">
+            <h2>Task Arguments</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.</p>
```
Comment (ghillert) [inline ui/src/app/tasks/task-launch/task-launch.component.html]: Lorem ipsum???

File: ui/src/app/tasks/task-launch/task-launch.component.html [hunk 3]
```diff
@@ -1,17 +1,120 @@
...
+                </td>
+              </tr>
+              </tbody>
+            </table>
+          </div>
+        </div>
+        <div class="col-md-12">
+          <div class="col-2-layout" formArrayName="params">
+            <h2>Task Properties</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt.</p>
```
Comment (ghillert) [inline ui/src/app/tasks/task-launch/task-launch.component.html]: Lorem ipsum???

**Comment by sabbyanandan** [comment]
Hi, @BoykoAlex. There are a few composed-task related code changes in this PR. Let us know if you have any thoughts.

**Comment by ghillert** [comment]
Created https://github.com/spring-cloud/spring-cloud-dataflow-ui/issues/723

**Comment by ghillert** [comment]
- Rebased (and resolved conflicts)
- Polished some wording
- Removed "Composed" task reference from create task screen
- merged with commit 611547552db898707da87cbcd11b6c5e5755c39e

**Pull request closed without merging**